.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description
+[Tell us about your new feature, improvement, or fix! If relevant, please supplement the PR with images.]
+
+
+## Checklist
+[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
+- [ ] [PEP8 compliance](https://www.python.org/dev/peps/pep-0008/)
```

Comment:
"Clean style in the spirit of PEP8" might be even better, especially considering PEP8 tells us to break it when we feel justified.